src/components/dialogs/inputs/input-hooks.js

```diff
@@ -513,17 +518,31 @@ export const useRegulatingTerminalValue = ({
     }, [inputForm.toggleClear]);
 
     useEffect(() => {
-        if (!voltageLevelOptionsPromise) return;
-
-        voltageLevelOptionsPromise.then((values) => {
-            setVoltageLevelOptions(
-                Array.from(values, (val) => val.voltageLevel).sort((a, b) =>
-                    a.id.localeCompare(b.id)
-                )
+        if (studyUuid && currentNodeUuid)
+            fetchVoltageLevelsIdAndTopology(studyUuid, currentNodeUuid).then(
```

Comment:
it will fetch those values several time if this component is included several times in a dialog
it's fixed with RegulatingTerminalForm which will replace this hook though

Passing options as props would fix it but I don't know if it's mandatory, since we don't use it multiple times in a non migrated dialog